packages/components/src/Accordion/useAccordion.js

```diff
@@ -22,20 +65,23 @@ export function useAccordion(props) {
 		...otherProps
 	} = useContextSystem(props, 'Accordion');
 
-	const accordionStore = useSubState((set) => ({
+	/** @type {import('zustand').UseStore<AccordionStore>} */
+	const accordionStore = useSubState((
```

Comment:
Interesting! This example is super helpful. It'll help me understand how to type (or try to type) components using `substate` 🎉